src/language/detect_language.py

```diff
@@ -59,6 +60,7 @@
     'Swift': ['swift'],
     'TypeScript': ['ts', 'tsx'],
     'Vue': ['vue'],
+    'XML': ['xml'],
```

Comment:
WE don't want to include XML files. Here is why: https://github.com/codersrank-org/repo_info_extractor/pull/130

Reply:
I see, ok :)